wcmc_components/lib/wcmc_components/version.rb

```diff
@@ -1,3 +1,3 @@
 module WcmcComponents
-  VERSION = '0.2.0'
+  VERSION = '0.2.1'
```

Comment:
I thought maybe this should be a minor version increment but I guess you're looking at it from perspective that this may be expected behaviour from the beginning so it makes sense this is a patch

Reply:
this PR has had another merged into it, so the release now includes allowing displaying methods in the table as well as fixing the bug, so bumped the version to 0.3.0 instead